src/main/java/seedu/address/model/Reminder.java

```diff
@@ -2,24 +2,34 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Objects;
 
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.model.reminder.ReminderStub;
+
+
 /**
  * Reminder object with description and dates remaining
  */
 public class Reminder {
 
     private HashMap<String, Integer> reminders;
     private HashMap<String, Integer> followup;
+    private ArrayList<ReminderStub> reminderArrayList;
```

Comment:
This ArrayList is causing the issue?

Reply:
According to Jerrold, it might be.

Reply:
Does it pass CI without this? I think its best to get @ReignOfComputer to resolve this though because this implementation is under his component.

Reply:
The Arraylist is my implementation but I used it as an example to get the UI up. I think he could remove this if he could find a replacement with his Hashmap to put into an ObservableList

Reply:
Let's figure it out in person when we meet tomorrow.

Reply:
Discussed with @ReignOfComputer. For now, I will temporarily disable the affected test cases and will let Jason resolve the logic problem.

Reply:
Okay fair enough. 👍